src/chat/infinite.js

```diff
@@ -64,6 +64,29 @@ async function createIfNotExists (doc, contentType = 'text/turtle', data = '') {
   return response
 }
 
+function desktopNotification (str) {
+  // Let's check if the browser supports notifications
+  if (!('Notification' in window)) {
+    alert('This browser does not support desktop notification')
```

Comment:
An alert is perhaps a bit too intrusive? I know this probably won't happen to most users, but perhaps you should tone it down to a `console.warn` until we have something better in place?

Reply:
When you're good with this, you can merge it as you please.

Reply:
done